aries-site/src/examples/templates/dashboards/content/RecentActivity.js

```diff
@@ -0,0 +1,142 @@
+      break;
+    case 'Firmware update failed':
+      title = 'Firmware update unsuccessful';
+      message =
+        count === 1
+          ? `Firmware update was unsuccessful for ${count} device.`
+          : `Firmware update was unsuccessful for ${count} devices.`;
+      break;
+    case 'Power on':
+      message =
+        count === 1
+          ? `${count} device was powered on.`
+          : `${count} devices were powered on.`;
+      break;
+    case 'Power off':
+      message =
+        count === 1
+          ? `${count} device was powered off.`
+          : `${count} devices were powered off.`;
+      break;
+    case 'Reset':
+      message =
+        count === 1
+          ? `${count} device was reset.`
+          : `${count} devices were reset.`;
+      break;
+    case 'Activate':
+      message =
+        count === 1
+          ? `${count} device was activated.`
+          : `${count} devices were activated.`;
+      break;
+
+    default:
+      break;
+  }
+  return { title, message };
+};
+
+export const RecentActivity = () => {
+  const data = MOCK_DATA;
+  const [activities, setActivities] = useState();
+
+  // Group like activities and create activity summary
+  useEffect(() => {
+    if (data) {
+      const nextActivities = data.activities.items.reduce(
+        (previous, current) => {
+          const { title, createdAt, selfUri } = current;
+          const next = { ...previous };
+          const count = previous[title]?.count + 1 || 1;
+          let mostRecent = createdAt;
+          let nextSelfUri = selfUri;
+
+          if (Date(createdAt) < Date(previous[title]?.mostRecent)) {
+            mostRecent = previous[title].mostRecent;
+            nextSelfUri = previous[title].selfUri;
+          }
+
+          const { title: adjustedTitle, message } = generateMessage(
+            title,
+            count,
+          );
+
+          next[title] = {
+            title: adjustedTitle,
+            count,
+            mostRecent,
+            message,
+            selfUri: nextSelfUri,
+          };
+
+          return next;
+        },
+        {},
+      );
+
+      setActivities(nextActivities);
+    }
+  }, [data]);
+
+  return (
+    <ActivityFeed title="Activity" footer={<FeedFooter />}>
+      {activities &&
+        Object.entries(activities)
+          .sort((a, b) => {
+            if (a[1].mostRecent > b[1].mostRecent) return -1;
+            if (a[1].mostRecent < b[1].mostRecent) return 1;
+            return 0;
+          })
+          .slice(0, 5)
+          .map(([, activity]) => (
+            <Activity
+              title={activity.title}
+              icon={<CircleInformation size="small" />}
+              message={activity.message}
+              timestamp={activity.mostRecent}
+              detailUrl={activity.selfUri}
+            />
+          ))}
+    </ActivityFeed>
+  );
+};
+
+const FeedFooter = () => (
+  <>
+    <Box />
```

Comment:
Why is this empty box necessary?

Reply:
CardFooter has default justify="between"

Reply:
Would it make more sense to add `justify="end"` to the CardFooter as opposed to adding an empty div to the DOM?

Reply:
Done.